src/_includes/partials/footer.html

```diff
@@ -0,0 +1,64 @@
+<footer>
+	<div>© 2021 lunch.dev</div>
```

Comment:
```suggestion
	<div><script type="text/javascript">document.write(new Date().getFullYear());</script> lunch.dev</div>
```
How do we feel about this (conceptually) for future maintenance?! I think the eleventy way of doing this would be to [use data files](https://www.11ty.dev/docs/data-js/#using-js-data-files), exporting a `currentYear` helper and using it here.

Reply:
If I may, I'd suggest removing this and replacing it with something "By the lunch dev community". Only because there's not an entity under which to copyright anything.

Which brings up an interesting point, I should find out how conferences handle rights. Because it's a little bit of a weird area. Is it a shared copyright? Can we get away with it being exclusively that of the author? Obviously we want to protect folks from plagiarism (especially with videos) but how do we do that? I'm not totally sure.

For those reasons I'd go with something friendly but "non-legal" for now"